wallet/src/wallet/mod.rs

```diff
@@ -209,11 +260,16 @@ impl<B: storage::Backend> Wallet<B> {
 
         db_tx.commit()?;
 
-        // TODO: Rescan blockchain
-
-        self.accounts.insert(account.account_index(), account);
+        // if it is the only one insert it directly to accounts
+        if self.accounts.is_empty() {
+            self.accounts.insert(account.account_index(), account);
+        }
+        // else it is unsynced as the other accounts have already processed blocks
+        else {
+            self.unsynced_account = Some((account.account_index(), account));
+        }
```

Comment:
These comments break the usual formatting of the `if..else` construct.How about putting them inside the curly braces?